unleashandroidsdk/src/test/java/io/getunleash/android/InspectionableCache.kt

```diff
@@ -0,0 +1,25 @@
+package io.getunleash.android
+
+import io.getunleash.android.cache.ToggleCache
+import io.getunleash.android.data.Toggle
+import io.getunleash.android.data.UnleashState
+
+/**
+ * This class is used to expose the cache for inspection during tests
+ */
+class InspectionableCache(
```

Comment:
Small nit, I believe you're looking for the word Inspectable, not Inspectionable
```suggestion
class InspectableCache(
```

Reply:
None of the two are recognized by my intellij spell checker but your suggestion sounds more natural

Reply:
[f5c0747](https://github.com/Unleash/unleash-android/pull/63/commits/f5c0747b73a5fce2a9a773bb6f20423fb49b6709)